### ISSUE TITLE
Disabled Searchableselect screenreader and tab when only using keyboard don't ignore the open button. 

### ISSUE THREAD

**Issue by LindaMeijering** [formkit/formkit#1380]
### Reproduction

https://formkit.link/5d040070cbade982ee2ba31faa2b690d

### Describe the bug

For this disabled input when I only use the keyboard to navigate they don't skip the input altogether but land on the chevron down and the screen reader reads "open".

### Environment

• OS: [e.g. iOS]
• Browser [e.g. chrome, safari]
• Version [e.g. 22]

**Comment by sashamilenkovic** [comment]
@LindaMeijering Happy to take a look at these! Issue #1379, #1381, #1382. If you could just update these playground links to where they're showing the inputs in question (not just the default text input), that would be great.

**Comment by sashamilenkovic** [comment]
@LindaMeijering When you say, "For this disabled input when I only use the keyboard to navigate they don't skip the input altogether". I'm assuming you are talking about using a screenreader as I cannot duplicate this happening without one (if this is not the case, feel free to skip everything else I'm about to say and please respond with more details of how I can reproduce this).

When using Apple's Voiceover, the input and the listbox are "focusable" by the screenreader, however they have the correct notation of "dimmed" (telling the user that the input is disabled). So, I don't see any issue there.

I understand you are using this Chrome extension: https://chromewebstore.google.com/detail/screen-reader/kgejglhpjiefppelpmljglcjbhoiplfn?hl=en-US&utm_source=ext_sidebar, but I am having a difficult time navigating to the disabled input in general with this screen reader. 

Nevertheless, perhaps you can provide more information as to why being able to navigate to these disabled inputs using a screen reader is an issue.

**Comment by LindaMeijering** [comment]
@sashamilenkovic I have tried using the Microsoft narrator and though the open button is read as disabled the input and listbox are not focusable. They are when using the apple screenreader?
